config.example.wikimedia.yaml

```diff
@@ -39,7 +39,7 @@ wikimedia.org: &wikimedia.org
     - path: projects/wikimedia.org.yaml
       options:
         <<: *default_options
-        pageviews:
+        metrics:
```

Comment:
I disagree, I think we don't really have a choice since we're adding a new endpoint to AQS.  Don't we need an RB config change for that anyway?  But even if we don't, it doesn't make too much sense to point something called pageviews to AQS in general.